### PR TITLE
cran fixes attempt 2

### PR DESCRIPTION
**Pull request by alkaZeltser**
After second attempt at submitting to CRAN we got the following comments:

```
Please do not start the description with "This package", "Tools for", 
package name, title or similar.
 
Please always explain all acronyms in the description text. -> VCF, SNP
 
If there are references describing the methods in your package, please 
add these in the description field of your DESCRIPTION file in the form
authors (year) <doi:...>
authors (year, ISBN:...)
or if those are not available: <https:...>
with no space after 'doi:', 'https:' and angle brackets for 
auto-linking. (If you want to add a title as well please put it in 
quotes: "Title")
 
You write information messages to the console that cannot be easily 
suppressed.
It is more R like to generate objects that can be used to extract the 
information a user is interested in, and then print() that object. 
Instead of print()/cat() rather use message()/warning() or 
if(verbose)cat(..) (or maybe stop()) if you really have to write text to 
the console. (except for print, summary, interactive functions)
-> R/apply-pgs.R
 
Please fix and resubmit.
```
Thus this PR contains
- updated DESCRIPTION with altered start, explained acronyms, and added reference citation
- the culprit print statement in `apply.polygenic.score` wrapped in `message`

- [x] I have read the [code review guidelines](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3187646/Code+Review+Guidelines) and the [code review best practice on GitHub check-list](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3189956/Code+Review+Best+Practice+on+GitHub+-+Check+List).

- [x] The name of the branch is meaningful and well formatted following the [standards](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3189956/Code+Review+Best+Practice+on+GitHub+-+Check+List), using \[AD_username (or 5 letters of AD if AD is too long)-\[brief_description_of_branch].

- [x] I have set up or verified the branch protection rule following the [github standards](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3190380/GitHub+Standards#GitHubStandards-Branchprotectionrule) before opening this pull request.

- [ ] I have added the changes included in this pull request to `NEWS` under the next release version or unreleased, and updated the date.

- [ ] I have updated the version number in `metadata.yaml` and `DESCRIPTION`.

- [ ] Both `R CMD build` and `R CMD check` run successfully.

## Testing Results

All unit tests PASS

### CODE REVIEW

**Comment by alkaZeltser** [comment]
The Description in DESCRIPTION sounds a bit wordy... might just be me. Taking suggestions!

**Review by forbiddenpersimmon — APPROVED** [review]

**Comment by alkaZeltser** [comment]
@dan-knight can you take a look please?

**Review by dan-knight — APPROVED** [review]
Looks like these CRAN issues are addressed. Did you run a Find in Files (or similar) for `print` statements to be sure you found them all? Easy for things to slip by these types of manual fixes.

**Comment by alkaZeltser** [comment]
> Looks like these CRAN issues are addressed. Did you run a Find in Files (or similar) for `print` statements to be sure you found them all? Easy for things to slip by these types of manual fixes.

I searched the repo through the github search bar, and also with grep: `grep -Rnw . -e "print"`
Is there another method you would recommend?

**Comment by dan-knight** [comment]
> Is there another method you would recommend?

Nope! All good - just some due diligence.
